dashboard/test/controllers/teacher_feedbacks_controller_test.rb

```diff
@@ -14,8 +14,8 @@ class TeacherFeedbacksControllerTest < ActionController::TestCase
     get :index
     assert_response :success
 
-    all_feedback_data = get_all_response_feedback_data
-    assert_equal 0, all_feedback_data.count
+    all_feedback_by_level = get_all_response_feedback_data
```

Comment:
I think this should probably be `all_feedbacks_by_level` to be consistent with the rest of the naming?